public/app/features/search/service/bluge.ts

```diff
@@ -69,8 +69,8 @@ export class BlugeSearcher implements GrafanaSearcher {
   }
 }
 
-const firstPageSize = 50;
-const nextPageSizes = 100;
+const firstPageSize = 5; //50;
```

Comment:
The UI expects the first query to return enough results to fill the screen, so this is weird -- but does make it easier to debug :)